src/js/App/RootApp/RootApp.js

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { Router } from 'react-router-dom';
 import { isBeta } from '../../utils';
 import chromeHistory from '../../utils/chromeHistory';
+import { FeatureFlagsProvider } from '../FeatureFlags';
 import IDPChecker from './IDPChecker';
 import ScalprumRoot from './ScalprumRoot';
 
 const RootApp = (props) => (
   <Router history={chromeHistory} basename={isBeta() ? '/beta' : '/'}>
-    <IDPChecker>
-      {/* <CrossRequestNotifier /> */}
-      <ScalprumRoot {...props} />
-    </IDPChecker>
+    <FeatureFlagsProvider>
```

Comment:
What's the benefit of having a extra component? We could directly import `UnleasFlagProvider` and have the `config` value be in constant file. Unless there's expected more props to be passed to this component in future (className, conditional tun on/off, etc.) I would rather keep it simple.

Reply:
I expect that more configuration will be required in a near future.

Reply:
I also want to keep the RootApp as simple as possible. We don't want file to have 500 lines like it used to have in chrome 1

Reply:
Then it would be better to have a component called `ChromeWrapper` with `UnleasFlagProvider`, `IDPChecker` and `ScalprumRoot`.

That way `RootApp` would set the Router and `ChromeWrapper` the other providers.

Reply:
The `RootApp` is meant to be such a wrapper.